support-lib/wasm/djinni_wasm.cpp

```diff
@@ -201,10 +187,18 @@ EM_JS(void, djinni_register_name_in_ns, (const char* prefixedName, const char* n
         ns[name] = Module[prefixedName];
 });
 
+void djinni_throw_native_exception(const std::exception& e) {
+    static auto throwCppException = em::val::module_property("djinni_throw_cpp_exception");
+    if (const auto* jsEx = dynamic_cast<const JsException*>(&e)) {
+        throwCppException(jsEx->cause());
```

Comment:
The val type has a throw() method, I believe you can do:
```suggestion
        jsEx->cause().throw();
```

Reply:
Thanks, this looks much nicer